.github/workflows/ci.yml

```
@@ -17,7 +17,10 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version-file: ./go.mod
+          # HACK(mafredri): The exampels and thirdparty library require Go 1.24
```

typo here and on line 33
```suggestion
          # HACK(mafredri): The examples and thirdparty library require Go 1.24
```